data/tlg0640/__cts__.xml

```diff
@@ -1,3 +1,3 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0640">
-  <ti:groupname xml:lang="lat">Alciphron Rhet., Soph., AD 2/3</ti:groupname>
+  <ti:groupname xml:lang="lat">Alciphron Rhet., Soph.</ti:groupname>
```

Comment:
I wonder if we shouldn't spell out what "Rhet." and "Soph." are?

Reply:
Probably. I also just looked this author up and the canonical authority name seems to be Alciphron, I think this is just from the TLG name, I should have changed it in the list.